docs/Guide/application-commands/application-command-registry/registering-context-commands.mdx

```diff
@@ -3,35 +3,34 @@ sidebar_position: 4
 title: Registering Context Commands
 ---
 
-On the page [Registering Chat Input Commands][rcic] we have learned how to register a Chat Input Command and how to
-assign options to those commands. However there is another another type of command, Context Menu Commands.
+Context Menu Commands are another kind of application command. They show up, as the name implies, in the context menus
+of either messages or users when you right click them.
 
-Context Menu Commands show up as the name implies, in the context menu. The specific menus are those when right clicking
-a user, and those when right clicking a message.
+| Message Context Menu                    | User Context Menu                    |
+| --------------------------------------- | ------------------------------------ |
+| ![](./message-context-menu-command.png) | ![](./user-context-menu-command.png) |
 
-Message Context Menu Commands look like this:
-
-![Message Context Menu Command Example](./message-context-menu-command.png)
+## How to register a Message Context Menu Command
 
-And User Context Menu Commands look like this:
+:::warning
```

Comment:
```suggestion
:::caution
```

Reply:
In my opinion, some details in the previous section are important enough that it should be a red caution for the people who skip straight to this page.